third_party/eigen3/BUILD

```diff
@@ -64,21 +64,15 @@ genrule(
         ":eigen_third_party_header_files",
     ],
     outs = ["include"],
+    tags = ["manual"],
     cmd = """
     mkdir $@
-    for f in $(locations @eigen_archive//:eigen_header_files) ; do
+    for f in $(SRCS); do
```

Comment:
@av8ramit I think you looked into eigen headers before?
Could you take a look?

Reply:
fwiw ive been using this in the gentoo ebuild and it outputs the files in the same place as the pip package. this part is just a small cleanup. Also the //tensorflow:install_headers dumps the eigen headers out too since they are transitively used by the rest. they output the same paths too.
I'll use this in gentoo for a while for some more testing then eventually we should make the pip package use this target instead of mangling the headers and external stuff in the build_pip_package.sh script so all installations have the same set of files.

Reply:
oh and the main difference between this way and the original way i used to install the headers (basically find -name "*.h") is that now the _internal.h and test headers are not included.